e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v2_test.js

```diff
@@ -3,13 +3,14 @@
 const config = require('../../../config.js');
 const mpScenario = 'ONE_V_TWO';
 const caseWorkerUser = config.hearingCenterAdminWithRegionId1;
```

Comment:
I would suggest using user for "caseWorkerUser" , "JudgeUser" if EA courts are enabled or not

Reply:
Done